modules/cnv-about-upgrading-cnv.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/upgrading-container-native-virtualization.adoc
+
+[id="cnv-about-upgrading-cnv_{context}"]
+= About upgrading {CNVProductName}
+
+If you enabled automatic updates when you installed {CNVProductName}, you
+will receive updates as they become available.
+
+.Additional information
+
+* In {CNVProductName} version 2.1, only _z-stream_ updates are available.
+** For example: {CNVProductName} 2.1.0 -> {CNVProductName} 2.1.1
+
+* Updates are delivered via the _Marketplace Operator_, which is deployed
+during {product-title} installation. The Marketplace Operator makes
+external Operators available to your cluster.
+
+* Upgrading does not interrupt virtual machine workloads.
+** Because workloads are uninterrupted, virtual machine Pods are not restarted
+or migrated during an upgrade. If you need to update the `virt-launcher` Pod,
+you must restart or live migrate it.
```

Comment:
you must restart or live migrate the virtual machine.

Reply:
Good catch, thanks! Fixed.